bundles/org.openhab.binding.jellyfin/README.md

```diff
@@ -0,0 +1,130 @@
+# Jellyfin Binding
+
+This binding allows connect to Jellyfin clients that supports remote control, it's build on top of the official Jellyfin kotlin sdk.
```

Comment:
Could you be a bit more verbose, so that somebody who hasn't heard about Jellyfin gets a clue what this is about?